crates/test/mock-consensus/src/builder.rs

```diff
@@ -5,12 +5,16 @@
 /// Most importantly, defines [`Builder::init_chain()`].
 mod init_chain;
 
-use {crate::TestNode, bytes::Bytes};
+use {
+    crate::{keyring::Keys, TestNode},
+    bytes::Bytes,
+};
 
 /// A buider, used to prepare and instantiate a new [`TestNode`].
 #[derive(Default)]
 pub struct Builder {
-    app_state: Option<Bytes>,
+    pub app_state: Option<Bytes>,
+    pub keys: Option<Keys>,
```

Comment:
NB: im avoiding defining a map, and the other keyring related interfaces, for now. i hope to save that for a follow-on PR.

Reply:
In this case it seems unclear what we're saving, not defining a map means having to define a custom Keys structure; if this were a BTreeMap<VerificationKey, SigningKey> there would be no extra structure to define and it would also be the data we actually want to use.

Reply:
e66628697 addresses this